src/main/java/gov/cabinetofice/gapuserservice/service/jwt/impl/CustomJwtServiceImpl.java

```diff
@@ -42,10 +43,10 @@ public boolean isTokenValid(final String customJwt) {
         }
 
 
-        return !isInBlacklist(customJwt);
+        return !isTokenInBlacklist(customJwt);
     }
 
-    private boolean isInBlacklist(final String customJwt) {
+    private boolean isTokenInBlacklist(final String customJwt) {
```

Comment:
This method exists in the JwtBlacklistService. Should we inject that service into here? My gut tells me that might be bad practice because of potential circular dependencies, and we should take the extremely minor code duplication hit, but curious to hear your thoughts

Reply:
Yeah, I dithered on this with Rachel's PR but think it's fine to leave it. I think the answer I would sway towards would be wrapping the repository in a DAO class and injecting that but thats a design pattern change and something we can potentially do if this problem arises again.